### DISCUSSION TITLE
How to back up my Netbox?

### DISCUSSION THREAD

**Discussion by manuelcalder0n** [netbox-community/netbox#6950] [Q&A] [answered]
Hello,

I'm trying to back up my netbox database in order to restore it in case of a data loss or a server failure. I have tried using pg_dump commands, but I'm not sure if it is the right way.

Could anyone help me? Has netbox any tool to perform the back up security copy? How would the recover process of installation be?

Thanks in advance.

Best regards,

Manuel.

**Comment by jeremystretch** [comment, marked as answer]
The complete will depend to a degree on the characteristics of your specific deployment, but you can find some general guidance on backing up the database and media files [here](https://netbox.readthedocs.io/en/stable/administration/replicating-netbox/). Hope that helps!

**Reply by manuelcalder0n** [reply]
So I asume that the only way is using pg_dump command. Thaks for your prompt response.

**Reply by jeremystretch** [reply]
NetBox uses PostgreSQL as its database, so you really have the entire PostgreSQL toolkit at your disposal. You can read more in [the PostgreSQL documentation](https://www.postgresql.org/docs/9.6/backup.html).

**Reply by manuelcalder0n** [reply]
Thank you Jeremy
